app/src/main/res/values-ru/sura_names.xml

```diff
@@ -0,0 +1,119 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+  <string-array name="sura_names">
+    <item>аль-Фатиха (Открывающая Книгу)</item>
```

Comment:
I verified with 2 translation. In the first one all letters was capitalized, but in the second one prefixes was lowercase. I'll check it again